src/components/v2/V2Create/forms/TokenForm/index.tsx

```diff
@@ -252,14 +273,60 @@ export default function TokenForm({
     formatDiscountRate(BigNumber.from(discountRate)),
   )
 
+  // Total tokens minted as a result of a 1 ETH contribution
+  const initialMintingRate = parseFloat(weight) ?? DEFAULT_ISSUANCE_RATE
+  const reservedRateDecimal = reservedRatePercent * 0.01
   // Tokens received by contributor's per ETH
   const initialIssuanceRate =
-    DEFAULT_ISSUANCE_RATE - reservedRatePercent * MAX_RESERVED_RATE
+    initialMintingRate - reservedRateDecimal * initialMintingRate
 
   return (
     <Form layout="vertical" onFinish={onTokenFormSaved}>
       <Space size="middle" direction="vertical">
         <div>
+          <div style={{ display: 'flex' }}>
```

Comment:
**suggestion** could we have this component in its own file?

Reply:
Yeah good call, done